docs/example-standalone-static-routing.toml

```diff
@@ -0,0 +1,63 @@
+bind_addr = "0.0.0.0"
+http_port = 5000
+grpc_port = 8831
+log_level = "info"
+
+[runtime]
+read_thread_num = 20
+write_thread_num = 16
+background_thread_num = 12
+
+[analytic]
+wal_path = "/tmp/ceresdb/"
+write_group_worker_num = 16
+replay_batch_size = 500
+max_replay_tables_per_batch = 1024
+write_group_command_channel_cap = 1024
+
+[analytic.storage]
+type = "Local"
+data_path = "/tmp/ceresdb/"
+
+[analytic.table_opts]
+arena_block_size = 2097152
+write_buffer_size = 33554432
+
+[analytic.compaction_config]
+schedule_channel_len = 16
+schedule_interval = "30m"
+max_ongoing_tasks = 16
+
+# Route&Shard: public
+[[static_route.topology.schema_shards]]
+schema = 'public'
+[[static_route.topology.schema_shards.shard_views]]
+shard_id = 0
+[static_route.topology.schema_shards.shard_views.endpoint]
+addr = '127.0.0.1'
+port = 8831
+
+# Route&Shard: test
+[[static_route.topology.schema_shards]]
+schema = 'test'
+[[static_route.topology.schema_shards.shard_views]]
+shard_id = 0
+[static_route.topology.schema_shards.shard_views.endpoint]
+addr = '127.0.0.1'
```

Comment:
Update this chapter [static_routing](https://docs.ceresdb.io/deploy/static_routing.html).